channel/src/main/java/com/synopsys/integration/alert/channel/azure/boards/distribution/AzureBoardsProcessorFactory.java

```diff
@@ -65,6 +71,14 @@ public AzureBoardsProcessorFactory(
         AzureWorkItemService workItemService = new AzureWorkItemService(azureHttpService);
         AzureWorkItemQueryService workItemQueryService = new AzureWorkItemQueryService(azureHttpService, apiVersionAppender);
 
+        installCustomFieldsIfNecessary(
+            organizationName,
+            distributionDetails.getProjectNameOrId(),
+            distributionDetails.getWorkItemType(),
+            new AzureProjectService(azureHttpService, apiVersionAppender),
+            new AzureProcessService(azureHttpService, apiVersionAppender)
+        );
```

Comment:
This was simply forgotten in the rewrite. Without this code we would never be able to find existing issues in Azure Boards.